### PR TITLE
Support Sensu Go 6.2

### PR DESCRIPTION
**Pull request by treydock**
# Pull Request Checklist

## Description

Main changes:

* Allow `agent.yml` to become authoritative by setting `sensu::agent::agent_managed_entity: true`
* Support PostgreSQL enabling round robin scheduling
* Allow use of `memberOf` with `sensu_ldap_auth`
* Add support for Ubuntu 20.04

### CODE REVIEW

File: spec/classes/api_spec.rb [hunk 1]
```diff
@@ -1,7 +1,9 @@
 require 'spec_helper'
 
 describe 'sensu::api', :type => :class do
-  on_supported_os.each do |os, facts|
+  on_supported_os({
+    supported_os: [{ 'operatingsystem' => 'RedHat', 'operatingsystemrelease' => ['7'] }]
```
Comment (ghoneycutt) [inline spec/classes/api_spec.rb:5]: Why did this change?

Comment (treydock) [reply spec/classes/api_spec.rb:5]: To speed up unit tests.  This and several other unit test files don't change based on OS tested, and we already did this pattern on a few other files before this pull request.  The classes that actually change based on OS facts still test all OSes defined in metadata.json.

Comment (ghoneycutt) [reply spec/classes/api_spec.rb:5]: Please add a comment with each that explains why this is happening.

**Review by ghoneycutt — APPROVED** [review]
Please merge after adding comments. All tests already passing.

File: spec/classes/api_spec.rb (hunk 1 above)
Comment (treydock) [reply spec/classes/api_spec.rb:6]: Comment added

**Pull request merged**